reference-architectures/deploy-arch-ibm-slz-vsi-quickstart.md

```diff
@@ -0,0 +1,135 @@
+---
+
+copyright:
+  years: 2023
+lastupdated: "2023-03-21"
+
+keywords:
+
+subcollection: secure-infrastructure-vpc
+
+authors:
+  - name: "Vincent Burckhardt"
+    email: "vincent.burckhardt@ie.ibm.com"
+
+# The release that the reference architecture describes
+version: 1.0
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: url #TODO
+
+docs: https://test.cloud.ibm.com/docs/secure-infrastructure-vpc
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/reference-architectures/vsi-pattern.png
+
+related_links:
+  - title: "Secure infrastructure on VPC for regulated industries - Standard variation"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-vsi-ra"
+    description: "A deployable architecture that is based on the IBM Cloud for Financial Services reference and that provides virtual servers in a secure VPC for your workloads."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# Secure infrastructure on VPC for regulated industries - QuickStart variation
+{: #vsi-ra-qs}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+The QuickStart variation of the Secure infrastructure on VPC for regulated industries deployable architecture creates a fully customizable Virtual Private Cloud (VPC) environment in a single region. The solution provides virtual servers in a secure VPC for your workloads. The QuickStart variation is designed to deploy quickly for demonstration and development.
+
+# Architecture diagram
+{: #ra-vsi-qs-architecture-diagram}
+
+![Architecture diagram for the QuickStart variation of Secure infrastructure on VPC for regulated industries](vsi-quickstart.drawio.svg "Architecture diagram of VSI on VPC for regulated industries deployable architecture"){: caption="Figure 1. QuickStart variation of Secure infrastructure on VPC for regulated industries" caption-side="bottom"}
```

Comment:
Diagram looks to have broken images within for the vsi's

Reply:
Will follow up with creator